scrapinghub/client/utils.py

```diff
@@ -106,6 +107,20 @@ def __init__(self, cls, client, key):
             self._proxy_methods(methods)
             self._wrap_iter_methods([method[0] for method in methods])
 
+        if issubclass(cls, MappingResourceType):
+            methods = ['ignore_fields', 'expire', 'save', 'liveget',
+                       ('iter', '__iter__')]
+            self._proxy_methods(methods)
+            # the following methods are special and should be set to class
```

Comment:
To apply the special methods we have to set it on class level, and enforce rewriting it for `__str__`/`__repr__` - they always exist in any class